source/demo-code/recipe-info-rough/recipePage.js

```diff
@@ -0,0 +1,40 @@
+var windowSize = 0;
+function myFunction() {
+    var x = document.getElementById("links");
+    if (x.style.display === "block") {
+      x.style.display = "none";
+    } else {
+      x.style.display = "block";
+    }
+
+    x = document.getElementById("search");
+    if (x.style.display === "block") {
+      x.style.display = "none";
+    } else {
+      x.style.display = "block";
+    }
+}
+window.addEventListener("resize", function(event) {
+    if(this.window.innerWidth > 800){
```

Comment:
i think you could implement this same behavior in a few lines with a media query